src/javascripts/components/airports.js

```diff
@@ -1,16 +1,17 @@
 const showAirports = (array) => {
   document.querySelector('#boards').innerHTML = '';
-  document.querySelector('#add-button').innerHTML = '<button class="btn btn-success btn-lg mb-4" id="add-airport-btn">Add An Airport</button>';
+  document.querySelector('#add-button').innerHTML = '<button class="btn" id="add-airport-btn">Add An Airport</button>';
```

Comment:
Was this intentional? If so that's fine, but the "Add An Airport" button does not look like a button until you hover over it.